op-txproxy/go.mod

```diff
@@ -0,0 +1,92 @@
+module github.com/ethereum-optimism/optimism/op-txproxy
```

Comment:
Please update the module 
```suggestion
module github.com/ethereum-optimism/infra/op-txproxy
```

Reply:
ahh nice catch, had this service in the monorepo initially